editions/tw5.com/tiddlers/system/tw5.com-styles.tid

```diff
@@ -197,3 +198,24 @@ div.content {
 .tc-btn-download:active {
 	background: #1475ff;
 }
+
+/* WikiText rules */
+
+.multi-columns,
+.fourcolumns {
+	display: block;
+	column-count: 4;
+	column-gap: 1em;
+}
+
+/* Tone down `code` elements */
+.no-code-border code {
```

Comment:
Rather than toning them down, I'd much rather that we lost the borders entirely, and just styled the table cells to look like code elements. These nested borders look awful:

Reply:
>These nested borders look awful:

That's very subjective. On small screens there is a large gap between the top and the bottom of the table. It's only on a PC where it looks strange. 

Hmmm, We may need a different setting for big and small screens :/  Some more testing will be needed

Reply:
>These nested borders look awful:

They would look much better, if https://github.com/Jermolene/TiddlyWiki5/pull/6912 would have been merged, which fixes those problems.

Reply:
did remove it from this PR and create a new issue, so the info isn't forgotten. The new issue is more generic. https://github.com/Jermolene/TiddlyWiki5/issues/7011